src/js/server/utils.js

```diff
@@ -108,12 +132,36 @@ gpii.express.user.utils.createNewUser = function (that, userData) {
     return promiseTogo;
 };
 
+/**
+ * gpii.express.user.utils.verifyPassword function
+ *
+ * Given an instance of our standard couch `userRecord`, and the clear text
+ * `password`, check to see if the password is valid for for the user.
+ *
+ * @param {Object} userRecord - Our standard internal user object stored in Couch.
+ * @param {String} password - Password to use to login/unlock user.
+ * @return {Boolean} - True if this is the correct password, otherwise false.
+ */
 gpii.express.user.utils.verifyPassword = function (userRecord, password) {
     var encodedPassword = gpii.express.user.password.encode(password,
         userRecord.salt, userRecord.iterations, userRecord.keyLength, userRecord.digest);
     return encodedPassword === userRecord.derived_key;
 };
 
+/**
+ * gpii.express.user.utils.unlockUser method
+ *
+ * Attempts to look up username using the current CouchDB view. (At the time
+ * of writing either username or email). And unlock their account using `password`.
+ * If the username and password match, the Couch userData record will be returned.
+ * Otherwise a standard error Object is returned.
+ *
+ * @param {gpii.express.user.utils} that - Utils component.
+ * @param {String} username - Username to use for record lookup.
+ * @param {String} password - Clear text password to validate record with.
+ * @return {Object} The userData record if the password is correct, otherwise
```

Comment:
either add backticks around "userData" or break up into the words, i.e. "user data".